v2/jwt/error.go

```diff
@@ -0,0 +1,26 @@
+package jwt
+
+var ErrNotValidNow = errExpiredType{}
+
+func errNotValidNow(underlyingErr error) error {
+
+	return errExpiredType{
+		underLyingErr: underlyingErr,
+	}
+}
```

Comment:
since it is private, this method seems like something we dont need? Just doing the struct init in `jwt.go` wo enough

Reply:
removed the error constructor